certified-connectors/Secib/apiDefinition.swagger.json

```diff
@@ -0,0 +1,755 @@
+            "x-ms-dynamic-values": {
+              "operationId": "GetCabinets",
+              "value-path": "id",
+              "value-collection": "result/items",
+              "value-title": "nom"
+            }
+          }
+        ]
+      }
+    },
+    "/api/v1/secib/Views/{name}/dynamic-schema": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "schema": {
+                  "type": "object"
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad Request",
+            "schema": {}
+          },
+          "401": {
+            "description": "Unauthorized request",
+            "schema": {},
+            "headers": {
+              "WWW-Authenticate": {
+                "description": "WWW-Authenticate",
+                "type": "string"
+              }
+            }
+          },
+          "403": {
+            "description": "Forbidden",
+            "schema": {}
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {}
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {}
+          },
+          "503": {
+            "description": "Service Unavailable",
+            "schema": {}
+          }
+        },
+        "summary": "Retrieving view's schema",
+        "tags": [
+          "Views"
+        ],
+        "operationId": "GetViewDynamicSchema",
+        "description": "Retrieve the dynamic schema of the GetView operation's response.",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "name",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Name of the view.",
+            "x-ms-summary": "View's name",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "name": "cabinet",
+            "in": "query",
+            "type": "string",
+            "description": "Identifier of the firm to work with.",
+            "x-ms-summary": "Firm's identifier"
+          }
+        ]
+      }
+    }
+  },
+  "definitions": {},
+  "parameters": {},
+  "responses": {},
+  "securityDefinitions": {
+    "oauth2_auth": {
+      "type": "oauth2",
+      "flow": "accessCode",
+      "authorizationUrl": "https://login.windows.net/common/oauth2/authorize",
+      "tokenUrl": "https://login.windows.net/common/oauth2/authorize",
+      "scopes": {
+        "values":"{API Permissions of the connector in Azure App Registrations}"
+      }
+    }
+  },
+  "security": [
+    {
+      "oauth2_auth": [
+        "{API Permissions of the connector in Azure App Registrations}"
+      ]
+    }
+  ],
+  "tags": [
+    {
+      "name": "Dossiers",
+      "description": ""
```

Comment:
@thomasBeckSecib Empty description can be removed until unless you have a specific reason to keep them.

Reply:
I removed empty descriptions.